[2025-03-04T06:26:09.228+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T06:26:09.248+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v01.get_name scheduled__2021-11-03T00:00:00+00:00 [queued]>
[2025-03-04T06:26:09.259+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v01.get_name scheduled__2021-11-03T00:00:00+00:00 [queued]>
[2025-03-04T06:26:09.259+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-04T06:26:09.276+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2021-11-03 00:00:00+00:00
[2025-03-04T06:26:09.288+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3072) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-04T06:26:09.289+0000] {standard_task_runner.py:72} INFO - Started process 3085 to run task
[2025-03-04T06:26:09.288+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v01', 'get_name', 'scheduled__2021-11-03T00:00:00+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/task_with_api_taskflow.py', '--cfg-path', '/tmp/tmplx9mo0d8']
[2025-03-04T06:26:09.291+0000] {standard_task_runner.py:105} INFO - Job 164: Subtask get_name
[2025-03-04T06:26:09.338+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v01.get_name scheduled__2021-11-03T00:00:00+00:00 [running]> on host e52f68431fc8
[2025-03-04T06:26:09.415+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v01' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2021-11-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-03T00:00:00+00:00'
[2025-03-04T06:26:09.416+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-04T06:26:09.419+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Naveen', 'last_name': 'kumar'}
[2025-03-04T06:26:09.457+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-04T06:26:09.458+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v01, task_id=get_name, run_id=scheduled__2021-11-03T00:00:00+00:00, execution_date=20211103T000000, start_date=20250304T062609, end_date=20250304T062609
[2025-03-04T06:26:09.584+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-04T06:26:09.594+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
